### ISSUE TITLE
Add Separate Field for Hashtags 

### ISSUE THREAD

**Issue by rogercavanagh** [fleetingnotes/fleeting-notes-flutter#538]
Hi Matt,

I would like to have separate input for hashtags that would be copied into Obsidian YAML, cf, the Source field. In Obsidian,  my practice is to include most hashtags in YAML and not as part of the text using the hash symbol.

Hashtags would still be allowed in the body of the note, but would only be copied to Obsidian YAML if the FN YAML field was empty.

I posted this already in Discord not appreciating the distinction between the app and the plug-in.

**Comment by matthewwong525** [comment]
Yep the error has been patched (as of today) see this issue (https://github.com/fleetingnotes/fleeting-notes-obsidian/issues/78) which tracks the problem for more info. I'll be releasing a new plugin version soon.

**Comment by rogercavanagh** [comment]
👍🏻  Regards,Roger________________Roger Cavanaghm: +44 7706 161844t: +44 20 829 2884














 From: Matthew Wong ***@***.***>Reply: fleetingnotes/fleeting-notes-flutter ***@***.***>Date: 23 March 2023 at 21:31:50To: fleetingnotes/fleeting-notes-flutter ***@***.***>CC: rogercavanagh ***@***.***>, Author ***@***.***>Subject:  Re: [fleetingnotes/fleeting-notes-flutter] Add Separate Field for Hashtags  (Issue #538)  

Yep the error has been patched (as of today) see this issue (fleetingnotes/fleeting-notes-obsidian#78) which tracks the problem for more info. I'll be releasing a new plugin version soon.



—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

[

{

***@***.***": "http://schema.org",

***@***.***": "EmailMessage",

"potentialAction": {

***@***.***": "ViewAction",

"target": "https://github.com/fleetingnotes/fleeting-notes-flutter/issues/538#issuecomment-1481940653",

"url": "https://github.com/fleetingnotes/fleeting-notes-flutter/issues/538#issuecomment-1481940653",

"name": "View Issue"

},

"description": "View this Issue on GitHub",

"publisher": {

***@***.***": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]
